client/my-sites/customer-home/cards/actions/wp-for-teams-quick-links/index.jsx

```diff
@@ -4,13 +4,10 @@
 import React from 'react';
```

Comment:
Another rename here, so feel free to ignore (previous `wp-for-teams-quick-links` has been deleted, and `wp-for-teams-quick-links-compact` has been renamed to `wp-for-teams-quick-links`).